public/pages/events.js

```diff
@@ -1,5 +1,35 @@
 // modal functionality
 
-toogleModal = (event) => {
-  document.getElementById("modal").classList.toggle("modal");
+// toogleModal = (event) => {  
+//   document.getElementById("modal").classList.toggle("modal");
+// };
+let book_url = document.getElementById("book-link").href;
+        let book_link = document.getElementById("book-link");
+        if(book_url !== location.href){
+             document.getElementById("modal").style.display = "none";
```

Comment:
in order to not duplicate the code please create the class and add event listener to it